odl/test/largescale/trafos/fourier_slow_test.py

```diff
@@ -27,33 +27,20 @@
 # --- pytest fixtures --- #
 
 
-impl_params = [never_skip('numpy'), skip_if_no_pyfftw('pyfftw')]
-impl_ids = [" impl = '{}'".format(p.args[1]) for p in impl_params]
-
-
 # bug in pytest (ignores pytestmark) forces us to do this this
+impl_params = [never_skip('numpy'), skip_if_no_pyfftw('pyfftw')]
 largescale = " or not pytest.config.getoption('--largescale')"
```

Comment:
Don't we have a utility for this? E.g. `skip_if_no_largescale`?

Reply:
We do, but how to use it here? I don't think I can wrap the `skipif` thing again in a `skipif`, at least not without getting ugly results in `ids`.

Reply:
Just tested, wrapping again doesn't work.